testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -120,14 +124,43 @@ def test_rmsd(self):
         assert_almost_equal(rmsd, 6.820321761927005, 5,
                             err_msg="RMSD calculation between 1st and last "
                             "AdK frame gave wrong answer")
-        #test mass_weighted
+        # test masses as weights
+        last_atoms_weight = self.universe.atoms.masses
+        A = self.universe.trajectory[0]
```

Comment:
This gives Timesteps for A and B, with the side effect of moving the frame cursor.

Do you actually want to do 
```python
A = self.universe.trajectory[0].positions.copy()
```
???